what's new.md

```diff
@@ -64,6 +64,7 @@
 - All NPM packages have `latest` and `preview` streams ([RaananW](https://github.com/RaananW))
 - Added New Tools Tab in the inspector (env texture and screenshot tools so far) ([sebavan](http://www.github.com/sebavan))
 - Moved to gulp 4, updated dependencies to latest ([RaananW](https://github.com/RaananW))
+- Bumped Monaco Editor to 0.18.1 and improved TypeScript compilation pipeline in the Playground ([sailro](http://www.github.com/sailro))

```

Comment:
This is not the right file (this one is for stable). This is the one to edit:
https://github.com/BabylonJS/Babylon.js/blob/master/dist/preview%20release/what's%20new.md